.gitignore

```diff
@@ -36,13 +37,24 @@ ehthumbs.db
 [Dd]esktop.ini
 $RECYCLE.BIN/
 
-# Local environment variables
-.env
-
-# Python Salesforce Functions
-**/__pycache__/
-**/.venv/
-**/venv/
-
-.vscode/
-.husky/
\ No newline at end of file
+NaSitDataExport.zip
```

Comment:
These extra files don't need to be committed to the repository. You can select specific changes to stage in vscode before committing and pushing them